packages/frontend/translations/de-DE.yaml

```diff
@@ -172,6 +172,9 @@ ui:
       title: 'Kontakte'
       none: Keine Kontakte vorhanden
       numOffline: '{num} offline'
+    actions:
```

Comment:
anything other than English is actually going to get obliterated by automation. Crowdin, an external service, manages all the translations, and will automatically open a PR with what translations are present at: https://crowdin.com/project/emberclear